package.json

```diff
@@ -8,6 +8,7 @@
   "typings": "dist/esm/index.d.ts",
   "unpkg": "dist/umd/pushin.min.js",
   "exports": {
+    ".": "./dist/umd/pushin.js",
```

Comment:
This will need to point to the same file as the "main" entry: "./dist/esm/pushin.js".

The umd build is provided for CDN usage only, since umd is minified and will be more difficult to debug in an application.